builder/virtualbox/ovf/config.go

```diff
@@ -87,15 +90,24 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 	errs = packer.MultiErrorAppend(errs, c.VBoxManagePostConfig.Prepare(&c.ctx)...)
 	errs = packer.MultiErrorAppend(errs, c.VBoxVersionConfig.Prepare(&c.ctx)...)
 
-	if c.SourcePath == "" {
+	// Get downloadable URL
+	downloadableURL, err := common.DownloadableURL(c.SourcePath)
+	notURL := !strings.HasPrefix(downloadableURL, "http://") && !strings.HasPrefix(downloadableURL, "https://")
```

Comment:
Comparing with the logic we use when downloading an ISO, it doesn't care if it's a file or an http url. Unless we don't want to allow file urls, I say just remove this check